components/NewPersonView/Layout.tsx

```diff
@@ -84,16 +84,18 @@ const Layout = ({ person, children }: Props): React.ReactElement => {
     });
 
   const secondaryNavigation = [
-    {
-      text: 'Add case note',
-      href: `/people/${person.id}/case-note`,
-    },
     {
       text: 'Add warning note',
       href: `/people/${person.id}/warning-notes/add?id=${person.id}`,
     },
   ];
 
+  if (person.contextFlag === 'A')
```

Comment:
Could we use a feature flag instead please? https://github.com/LBHackney-IT/lbh-social-care-frontend/blob/main/.github/CONTRIBUTING.md#react-hook

Reply:
this isn't supposed to be a feature flag - we _want_ to show that link in all conditions on an adult person

Reply:
As we saw last time with the `AddForm` component (I believe) this will quickly get too messy and either need to be rewritten or forms will accidentally be released.

Also it is a mess to debug, we need to be using the flags on the forms `isViewableByAdults` and `isViewableByChildrens`

Reply:
Ahhhh sorry @jhackett1, I get it now! 🤦

Reply:
@JohnFarrellDev this isn't one of our flexible forms - this is the legacy form that we're taking out. there's no flags to check for